miso-web/src/main/webapp/scripts/warning_library.js

```diff
@@ -22,23 +22,21 @@
  */
 
 WarningTarget.library = {
-  headerWarnings: function(library){
-    var warnings = [];
-    warnings = Warning.addWarnings([
-      [library.subprojectPriority, 'Belongs to high priority subproject \'' + library.subprojectAlias + '\''],
-      [parseFloat(library.volume) < 0, 'This library has a negative volume!'],
-      [library.identityConsentLevel === 'Revoked', 'Donor has revoked consent']
-      ], warnings);
-    return Warning.generateHeaderWarnings(warnings);
-  },
-  tableWarnings: function(data, type, library){
-    var warnings = [];
-    warnings = Warning.addWarnings([
-      [library.subprojectPriority, 'PRIORITY (' + library.subprojectAlias + ')', "info"],
-      [parseFloat(library.volume) < 0, '(NEGATIVE VOLUME)'],
-      [library.identityConsentLevel === 'Revoked', '(CONSENT REVOKED)']
-      ], warnings);
-    return Warning.generateTableWarnings(data, warnings);
-  },
-  
+  getWarnings: function(library) {
+    return [{
```

Comment:
I had expected that this would warn about low-quality library, but it looks like that's never been there. What do you think about adding it?

Reply:
Adding